apps/approval/dashboard/views.py

```diff
@@ -104,7 +93,7 @@ def approve_application(request):
                 membership.save()
             else:
                 membership = Membership()
-                membership.username = user.ntnu_username
+                membership.username = user.email
```

Comment:
This should be user-id if it is something that isn't the email, but this is actually a bit of a assumption-error / issue that we might have been giving out memberships to new master-students without NTNU-accounts 🤔